assignments/prototype-refactor.js

```diff
@@ -7,3 +7,138 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+  
```

Comment:
Perfect use of your first commit. This is a significant change to the file that we want to record and you've done that perfectly.